pingcap/pd
# use gzip_handler of  dashboard

### What problem does this PR solve? 

Add gzip support for dashboard uiserver.

### What is changed and how it works?


### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

1. check the response content-encoding and size in the browser 
2. using `curl` test request resource with or without the `Accept-Encoding: gzip, deflate, br` header

related to https://github.com/pingcap-incubator/tidb-dashboard/pull/514
